lib/src/container/mod.rs

```diff
@@ -308,21 +325,31 @@ impl<'a> ManifestDiff<'a> {
 
 impl<'a> ManifestDiff<'a> {
     /// Prints the total, removed and added content between two OCI images
-    pub fn print(&self) {
+    pub fn print(&self) -> ExportedManifestDiff {
         fn layersum<'a, I: Iterator<Item = &'a oci_spec::image::Descriptor>>(layers: I) -> u64 {
             layers.map(|layer| layer.size() as u64).sum()
         }
         let new_total = self.to.layers().len();
-        let new_total_size = glib::format_size(layersum(self.to.layers().iter()));
+        let new_total_size = layersum(self.to.layers().iter());
+        let new_total_size_str = glib::format_size(layersum(self.to.layers().iter()));
         let n_removed = self.removed.len();
         let n_added = self.added.len();
         let removed_size = layersum(self.removed.iter().copied());
         let removed_size_str = glib::format_size(removed_size);
         let added_size = layersum(self.added.iter().copied());
         let added_size_str = glib::format_size(added_size);
-        println!("Total new layers: {new_total:<4}  Size: {new_total_size}");
+        println!("Total new layers: {new_total:<4}  Size: {new_total_size_str}");
         println!("Removed layers:   {n_removed:<4}  Size: {removed_size_str}");
         println!("Added layers:     {n_added:<4}  Size: {added_size_str}");
+        let manifest_diff = ExportedManifestDiff {
```

Comment:
See https://doc.rust-lang.org/std/keyword.return.html - we don't need explicit `return` in Rust in this situation.  We can just do:

```
ExportedManifestDiff { ... }
``

(But if we follow my first suggestion, we don't need this at all)

Reply:
I assume that "But if we follow my first suggestion, we don't need this at all" refers to moving the members of the `ExportedManifestDiff` directly into the `ManifestDiff` struct?